modules/ROOT/pages/migration-from-froala.adoc

```diff
@@ -292,34 +293,35 @@ The following examples show a custom toolbar layout in Froala and the same layou
   tinymce.init({
     selector: '#tiny_custom_toolbar',
     plugins: 'lists',
-    toolbar: 'undo redo | formatselect | bold italic
+    toolbar: `undo redo | formatselect | bold italic
       | alignleft aligncenter alignright alignjustify
-      | numlist bullist | outdent indent',
+      | numlist bullist | outdent indent`,
```

Comment:
You can't use ES6 or above syntax in TinyMCE 5 documentation sorry, as it must work with IE 11. As such, this should either be collapsed to a single line or broken up over a couple lines (so as to prevent a scrollbar). e.g
```
    toolbar: 'undo redo | formatselect | bold italic ' +
      '| alignleft aligncenter alignright alignjustify '+
      '| numlist bullist | outdent indent',
``

Reply:
ok, I changed it to a concatenation of strings as in your example